docs/async/async-csharp-vb.md

```diff
@@ -2,7 +2,7 @@
 
 By [Phillip Carter](https://github.com/cartermp)
 
-Async programming in C# and VB share a language-level asynchronous programming model which allows for easily writing asynchronous code without having to juggle callbacks or conform to a library which supports asynchrony. It follows what is known as the [Task-based Asynchronous Pattern (TAP)](https://msdn.microsoft.com/en-us/library/hh873175%28v=vs.110%29.aspx).
```

Comment:
We should change all the mentions to VB to Visual Basic, as we did with the titles.